airflow/www/templates/airflow/dags.html

```diff
@@ -20,6 +20,39 @@
 {% extends base_template %}
 {% from 'appbuilder/loading_dots.html' import loading_dots %}
 
+{%- macro verbose_ordering_name(ordering) -%}
+    {{ 'descending' if ordering == 'desc' else ('ascending' if ordering == 'asc' else ordering) }}
+{%- endmacro -%}
+
+{%- macro sorted_th_link(name, attr_name) -%}
+   {% set request_ordering = (request.args.get('orderBy', 'unsorted')) %}
+   {% set curr_ordering = ('unsorted' if request.args.get('sortBy') != attr_name else request_ordering ) %}
+   {% set new_ordering = ('asc' if request.args.get('sortBy') != attr_name or curr_ordering != 'asc' else 'desc' ) %}
+   <a href="{{ url_for('Airflow.index',
+                       status=request.args.get('status', 'all'),
+                       search=request.args.get('search', None),
+                       tags=request.args.get('tags', None),
+                       sortBy=attr_name,
+                       orderBy=new_ordering
+                       ) }}"
+   class="js-tooltip"
+   role="tooltip"
+   title="Press to sort by {{ verbose_ordering_name(new_ordering) }} {{ attr_name }}."
```

Comment:
One last nit… Can we make this tooltip less verbose? The "Press to" can be inferred by the interaction (hover) that invokes it.
```suggestion
   title="Sort by {{ verbose_ordering_name(new_ordering) }} {{ attr_name }}."
```

Reply:
Done!

Reply:
Appears that this suggestion didn't get applied.